pom.xml

```diff
@@ -312,6 +312,17 @@
             <id>release</id>
             <build>
                 <plugins>
+                <plugin>
+                         <groupId>org.sonatype.plugins</groupId>
+                        <artifactId>nexus-staging-maven-plugin</artifactId>
+                        <version>1.6.3</version>
+                        <extensions>true</extensions>
+                        <configuration>
+                            <nexusUrl>https://oss.sonatype.org/</nexusUrl>
+                            <serverId>ossrh</serverId>
+                            <autoReleaseAfterClose>${autorelease}</autoReleaseAfterClose>
```

Comment:
you're missing the property, and indentation is a bit off.  I would default it to false, since we aren't using this.